### PR TITLE
Update CHANGELOG with v2.21.0 changes

### PR DESCRIPTION
**Pull request by dataclouder**
Update `CHANGELOG.md` for upcoming release

Please check [https://github.com/dataclouder/go-vcloud-director/blob/release-2.21.0/CHANGELOG.md](https://github.com/dataclouder/go-vcloud-director/blob/release-2.21.0/CHANGELOG.md) for correct rendering

### CODE REVIEW

**Review by adambarreiro — APPROVED** [review]
LGTM

**Review by lvirbalas — APPROVED** [review]
What a big list!

**Review by adezxc — APPROVED** [review]

**Review by Didainius — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,99 @@
...
+  `DistributedFirewallRule.Update`, `.DistributedFirewallRuleDelete` to manage NSX-T Distributed
+  Firewall Rules one by one (opposed to managing all at once using `DistributedFirewall`) ([#587](https://github.com/vmware/go-vcloud-director/pull/587))
+* Added method `Vdc.CreateVappFromTemplate` to create a vApp from a vApp template containing one or more VMs ([#588](https://github.com/vmware/go-vcloud-director/pull/588))
+* Added method `Vdc.CloneVapp` to create a vApp from another vApp ([#588](https://github.com/vmware/go-vcloud-director/pull/588))
+* Added method `VApp.DiscardSuspendedState` to take a vApp out of suspended state ([#588](https://github.com/vmware/go-vcloud-director/pull/588))
+
+### IMPROVEMENTS
+* `ExternalNetworkV2` now supports IP Spaces on VCD 10.4.1+ with new fields `UsingIpSpace` and
+  `DedicatedOrg` ([#579](https://github.com/vmware/go-vcloud-director/pull/579))
+* Add a new function `WithIgnoredMetadata` to configure the `Client` to ignore specific metadata entries
```
Comment (lvirbalas) [inline CHANGELOG.md:76]:
```suggestion
* Added a new function `WithIgnoredMetadata` to configure the `Client` to ignore specific metadata entries
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,99 @@
...
+  in all non-deprecated metadata CRUD methods ([#581](https://github.com/vmware/go-vcloud-director/pull/581))
+* NSX-T ALB Virtual Service supports IPv6 Virtual Service using field`IPv6VirtualIpAddress` in
+  `types.NsxtAlbVirtualService` for VCD 10.4.0+ ([#582](https://github.com/vmware/go-vcloud-director/pull/582))
+* Add field `EnableDualSubnetNetwork` to enable Dual-Stack mode for Org VDC networks in
```
Comment (lvirbalas) [inline CHANGELOG.md:80]:
```suggestion
* Added field `EnableDualSubnetNetwork` to enable Dual-Stack mode for Org VDC networks in
```

**Review by lvirbalas — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dataclouder) [reply CHANGELOG.md]: changed

File: CHANGELOG.md (hunk 2 above)
Comment (dataclouder) [reply CHANGELOG.md]: changed

**Pull request merged**